docs/source/guide/release_notes.md

```diff
@@ -12,10 +12,69 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
     <i class='ent'></i> 
     The release notes for Label Studio Community Edition is available on the <a href="https://github.com/heartexlabs/label-studio/releases"> Label Studio GitHub repository</a>.
 
+
+## Label Studio Enterprise 2.3
+
+This section highlights the new features, enhancements, and bug fixes in Label Studio Enterprise 2.3.
+
+### New features and enhancements 
+
+Label Studio Enterprise 2.3 introduces the following new features and enhancements.
+
+- Streamline your project creation workflow by duplicating projects: Enable users to [duplicate projects](manage_users.html#More-menu-to-Pin-or-Unpin-and-Duplicate-projects) to a target workspace optionally importing the settings, tasks, and annotations. This feature allows administrators and managers to quickly spin up new projects and ensure consistency by duplicating existing projects in their workspace. You can choose to duplicate a project by settings only, both settings and data, or settings, data and all annotations. If choosing the latter, users can also decide whether to make annotations **Ground Truth**. To duplicate a project, navigate to project's **Settings** >> **More** menu, choose a destination for the new project, decide which attributes you would like to duplicate, and update the new project name and description [DEV-1202].
+- Pin and create shared view of relevant projects in your workspace: Allow users to [pin or unpin projects](manage_users.html#More-menu-to-Pin-or-Unpin-and-Duplicate-projects) to more easily filter and find projects of interest. This feature allows you to pin and create a shared view of relevant projects in your workspace. Administrators now have the ability to pin and unpin projects for quicker reference by team members, whether they are projects the team is currently working on, or projects of interest. All team members will have a view of pinned projects to collaborate on in their workspace [DEV-2069].
+- Zoom, align, and pixelate images for annotation: Annotators can zoom and expand images they are labeling within the viewport of the task, set the image to be positioned vertically (top, center, bottom) and horizontally (left, center, right) to find the most convenient position. Once an image is zoomed in, they can also choose to pixelate the image if it is blurry for more precise annotations [DEV-2504].
```

Comment:
is there a link to rich description here?

Reply:
No.